templates/scenarios/csharp/sso-tab/teamsapp.local.yml.tpl

```diff
@@ -5,15 +5,15 @@ version: 1.0.0
 provision:
   - uses: aadApp/create # Creates a new AAD app to authenticate users if AAD_APP_CLIENT_ID environment variable is empty
     with:
-      name: {{appName}}-aad # Note: when you run configure/aadApp, the AAD app name will be updated based on the definition of manifest. If you don't want to change the name, ensure the name in AAD manifest is same with the name defined here.
+      name: {{appName}}-aad # Note: when you run aadApp/update, the AAD app name will be updated based on the definition of manifest. If you don't want to change the name, ensure the name in AAD manifest is same with the name defined here.
```

Comment:
of manifest -> in manifest
ensure -> make sure
same -> the same

Reply:
thanks, updated